src/element/atropos-element.js

```diff
@@ -0,0 +1,115 @@
+/* eslint-disable no-restricted-globals */
+import AtroposCore from '../../build/esm/atropos.esm.js';
```

Comment:
Rename this class import to Atropos. Import path must be different, look at other components